src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -1384,7 +1385,7 @@ TEST_P(OVClassLoadNetWorkDoNotReturnDefaultHintTest, LoadNetworkDoNotReturnDefau
     if (target_device.find("AUTO") != std::string::npos) {
```

Comment:
What is the hardcode in the shared part???

Reply:
@ilya-lavrenov As we agreed before, MULTI is the only one which is with THROUGHPUT as default hint, actually AUTO is the only plugin which implemented this requirement.  If your recent PRs have implemented the requirement CVS-98248 (latency as default hint) for other plugins?